# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xosview/linux-header.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- xosview-1.8.3/linux/serialmeter.cc.vanilla	2007-02-04 15:05:55.000000000 +0000
+++ xosview-1.8.3/linux/serialmeter.cc	2007-02-04 15:06:06.000000000 +0000
@@ -45,7 +45,9 @@
 #endif
 #endif
 #include <linux/serial.h>
-#include <linux/serial_reg.h>
+
+#define UART_LSR        5       /* In:  Line Status Register */
+#define UART_MSR        6       /* In:  Modem Status Register */
 
 SerialMeter::SerialMeter( XOSView *parent, Device device )
   : BitMeter( parent, getTitle(device), "LSR bits(0-7), MSR bits(0-7)", 16){
